packages/reporters/cli/src/formatCodeFrame.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import {codeFrameColumns} from '@babel/code-frame';
+
+import type {DiagnosticCodeFrame} from '@parcel/diagnostic';
+
+export default function formatCodeFrame(
+  codeframe: DiagnosticCodeFrame
+): string {
+  if (codeframe.codeHighlights.length === 0) {
+    return 'Could not create codeframe, no highlights defined.';
+  }
+
+  // TODO: Support multiple code highlights
+  let highlight = Array.isArray(codeframe.codeHighlights)
+    ? codeframe.codeHighlights[0]
+    : codeframe.codeHighlights;
+
+  let result = codeFrameColumns(
+    codeframe.code,
```

Comment:
what would happen if this were not JS - like CSS or something?

Reply:
than codeframe.code should be css. In theory a codeframe isn't language specific.

The syntax highlighting of babel/code-frame is JS specific though